CCPAPI/PAPI/CMakeLists.txt

```diff
@@ -0,0 +1,72 @@
+##########################################################################
+#                                                                        #
+#                               PAPI                                     #
+#                                                                        #
+#  This program is free software; you can redistribute it and/or modify  #
+#  it under the terms of the GNU Library General Public License as       #
+#  published by the Free Software Foundation; version 2 or later of the  #
+#  License.                                                              #
+#                                                                        #
+#  This program is distributed in the hope that it will be useful,       #
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of        #
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the          #
+#  GNU General Public License for more details.                          #
+#                                                                        #
+#          Copyright 2020 Paul RASCLE www.openfields.fr                  #
+#                                                                        #
+##########################################################################
+
+cmake_minimum_required(VERSION 3.0)
```

Comment:
This should be in the top-level cmake, not in each of the files.

We have moved to requiring 3.10.